### PR TITLE
highlight current page

### PR DESCRIPTION
**Pull request by Mankavelda**

### CODE REVIEW

**Comment by elhmn** [comment]
Nice work again, I left a [comment](https://github.com/osscameroon/camerapps.com/pull/44#issuecomment-1119384729) on the other PR. I think once you fixed it on the other PR you could try to apply the same patch to this one 👌

File: src/views/partials/pagination.hbs [hunk 1]
```diff
@@ -31,7 +31,18 @@
                         <li class="page-item"><a href="{{@root.fullUrl}}&page={{.}}" class="page-link">{{ . }}</a></li>
                     {{/if}}
                 {{else}}
-                    <li class="page-item"><a href="{{@root.fullUrl}}?page={{.}}" class="page-link">{{ . }}</a></li>
+                {{#if (portContains @root.fullUrl "3000") }}
```
Comment (elhmn) [inline src/views/partials/pagination.hbs:34]: This won't work in production, as the fullUrl  will be something different and probably won't use a port

**Review by elhmn — CHANGES_REQUESTED** [review]

**Review by elhmn — CHANGES_REQUESTED** [review]
I tested the implementation and it does not seem to work, if we change the port. Could you test locally and let me know ?

File: src/helpers/handlebars.ts [hunk 2]
```diff
@@ -44,3 +44,16 @@ export const repoName = (v1: string): string => {
...
+  return a.slice(-1).includes(b);
+}
+
+
+export const portContains = (a: string, b:string): boolean => {
+  return a.includes(b);
+}
+
+export const not = (a: string): boolean=> {
+  return (!(a));
```
[the remark this answers is not in the document]
Comment (Mankavelda) [reply src/helpers/handlebars.ts:58]: Thanks @elhmn. i will make the changes right away!!!

**Pull request closed without merging**